ddc_nft_registry/ddc_nft_registry/attachment/entity.rs

```diff
@@ -0,0 +1,31 @@
+//! The data structure of Attachments.
+
+use ink_storage::traits::{PackedLayout, SpreadLayout};
+use scale::{Decode, Encode};
+use ink_prelude::string::String;
+
+use crate::ddc_nft_registry::contract_fee::{SIZE_ACCOUNT_ID, SIZE_INDEX, SIZE_NFT_ID, SIZE_PER_RECORD, SIZE_PROOF};
+
+pub type NftId = String;
+pub type AssetId = String;
+pub type Proof = String;
+
+#[derive(Clone, PartialEq, Encode, Decode, SpreadLayout, PackedLayout)]
+#[cfg_attr(feature = "std", derive(Debug, scale_info::TypeInfo))]
+pub struct Attachment {
+    pub owner_id: ink_env::AccountId,
```

Comment:
(After some macro magic, it will be defined as something like this in `mod ddc_nft_registry`)

    type AccountId = <<DdcBucket as ::ink_lang::ContractEnv>::Env as ::ink_env::Environment>::AccountId;